Week2/homework/exercise3.js

```diff
@@ -0,0 +1,13 @@
+"use strict"
+
+var myRecipe = {
+    name: "Brigadeiro",
+    serving: "30",
+    ingredients: ["condensed milk ", 
+    "dark chocolate ", 
+    "butter "],
+};
```

Comment:
👍 for using `use strict` and picking Brigadeiro(I like it too) but 👎 for using `var` instead of `let` or `const` keyword. Also, the indentation of the `ingredients` array is not the best. You will learn in JS2 why using `let` or `const` is better than using `var`

Reply:
Brigadeiros are awesome, they never should be at the same place as a var. Shame on me.